presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/TableAccessControlRule.java

```diff
@@ -22,31 +22,38 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
+import static io.prestosql.plugin.base.security.AccessControlRuleHelper.mergeRegexAndSet;
 import static java.util.Objects.requireNonNull;
 
 public class TableAccessControlRule
 {
     private final Set<TablePrivilege> privileges;
     private final Optional<Pattern> userRegex;
+    private final Optional<Pattern> groupRegex;
     private final Optional<Pattern> schemaRegex;
     private final Optional<Pattern> tableRegex;
 
     @JsonCreator
     public TableAccessControlRule(
             @JsonProperty("privileges") Set<TablePrivilege> privileges,
             @JsonProperty("user") Optional<Pattern> userRegex,
+            @JsonProperty("users") Optional<Set<String>> users,
```

Comment:
I am not sure if want these two. Notice that `users` could expressed as `user` with `user1|user2|user3`. Same for groups. Can you please remind me the motivation behind this?

Reply:
This was requested by some Starburst Presto users, with the motivation that JSON is less error-prone when it comes to unexpected special characters appearing in usernames, which could lead to lax resource control.